faqs/emergency-shutdown.md

```diff
@@ -0,0 +1,130 @@
+# Emergency Shutdown FAQ
+
+## What is an Emergency Shutdown?
+
+One important security feature of Dai is called Emergency Shutdown, formerly known as Global Settlement. This crucial security feature allows the system to shut down and make underlying collateral available for redemption by [Dai](https://github.com/makerdao/community/blob/master/faqs/dai.md) holders and CDP owners.
+
+## What happens during an Emergency Shutdown?
+
+1. Emergency Shutdown is activated:
+If MKR voters believe that the system is subject to a serious attack, or if an Emergency Shutdown is scheduled as part of a technical upgrade, they can activate an Emergency Shutdown. This stops CDP creation and freezes the Price Feed.
+
+2. Collateral Claims are processed:
```

Comment:
ordered list items should be repeating numbers: https://github.com/makerdao/community/blob/master/contributing/style-guide.md

Reply:
done